Dockerfile

```diff
@@ -172,9 +172,11 @@ RUN for _ in {1..10}; do su - amavis -c "razor-admin -create" ; sleep 3 ; \
 # ––– Fail2Ban, DKIM & DMARC ––––––––––––––––––––
 # –––––––––––––––––––––––––––––––––––––––––––––––
 
-COPY target/fail2ban/jail.conf /etc/fail2ban/jail.conf
-COPY target/fail2ban/filter.d/postfix-sasl.conf /etc/fail2ban/filter.d/postfix-sasl.conf
-RUN mkdir /var/run/fail2ban
+COPY target/fail2ban/jail.local /etc/fail2ban/jail.local
+RUN ln -s /var/log/mail/mail.log /var/log/mail.log && \
```

Comment:
Just styling: We use

``` DOCKERFILE
RUN \
  do_something_command \
  do_some_more_command
```

i.e. use the `\` first when multi-line `RUN` statements appear:)

Reply:
Oh, I must have overlooked that, thx.

Reply:
My mistake was, that I looked at this example directly above my change, which seems also not correct then? https://github.com/docker-mailserver/docker-mailserver/blob/34c6e46f68f5de60ac08f815252f4e1632eb78b0/Dockerfile#L167-L169

Reply:
I've fixed both styling issues.